.scripts/automation_init.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
-npm install -g @microsoft/rush@5.75.0
+npm install -g @microsoft/rush
```

Comment:
@dw511214992: Why did you unpin the version of `@microsoft/rush`?  It's usually better to pin a specific version, so our scripts do not break if there are breaking changes in new versions.

If you need to upgrade to a newer version of rush, I recommend just updating the version instead of unpinning.  So you could change this to `@microsoft/rush@5.82.1` to use the current latest version.

CC: @ckairen, @weshaggard